### ISSUE TITLE
Auto loop around tabs, like someone pressing ctrl+tab, on vscode start

### ISSUE THREAD

**Issue by linonetwo** [gitbito/bitoai#11]
Type: <b>Bug</b>

With this extension enable, this happened all the time when a workspace have multiple tabs opened previously.

Extension version: 1.0.130
VS Code version: Code 1.76.0 (92da9481c0904c6adfe372c12da3b7748d74bdcb, 2023-03-01T10:23:45.993Z)
OS version: Darwin x64 22.3.0
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz (16 x 2300)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|5, 5, 4|
|Memory (System)|16.00GB (1.73GB free)|
|Process Argv|--crash-reporter-id 01181e41-ddad-4852-9321-689a284c434e|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012:30540252
pynewext54:30669237
azure-dev_surveyone:30548225
vscccc:30610679
pyindex848:30662994
nodejswelcome1:30587005
3biah626:30602489
pyind779:30671433
f6dab269:30613381
pythonsymbol12:30671437
2i9eh265:30646982
6233i204:30672705
vsctsb:30662444
pythonb192:30669360

```

</details>

**Comment by pawelhoros** [comment]
yup, same here, macOS. 
When VS Code opened in full screen, everything is switching in loop.

**Comment by iwxyi** [comment]
same in Windows10

**Comment by bitoandy** [comment]
We have identified the issue and the fix will be available in the next release.

**Comment by bitoandy** [comment]
Hi @linonetwo , @pawelhoros , @iwxyi,

Please update to the latest build as this issue is fixed in the latest build of 1.0.132 [https://marketplace.visualstudio.com/items?itemName=Bito.Bito]
